packages/untp-test-suite/src/templates/mapper.ts

```diff
@@ -0,0 +1,33 @@
+import Handlebars from 'handlebars';
+import fs from 'fs/promises';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import { TestSuiteResult } from '../core/types/index.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+Handlebars.registerHelper('jsonArray', (array: object) => {
+  return JSON.stringify(array);
+});
+
+Handlebars.registerHelper('replace', (input: string, target: string, replacement: string) => {
+  const regex = new RegExp(target, 'g');
+  return input.replace(regex, replacement);
+});
+
+export async function templateMapper(templateName: string, testSuiteResult: TestSuiteResult) {
```

Comment:
Should we check `testSuiteResult` value if it exists or not?

Reply:
This project uses Typescript, so I don't think this function will execute if any of its parameters or nested fields are missing. Except if you call the built file (.js) of this file.